src/main/java/project/controllers/DeleteCarsController.java

```diff
@@ -0,0 +1,24 @@
+package project.controllers;
+
+import java.io.IOException;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import project.lib.Injector;
+import project.service.CarService;
+
+public class DeleteCarsController extends HttpServlet {
```

Comment:
```suggestion
public class DeleteCarController extends HttpServlet {
```
same for all DeleteControllers